docs/articles/blog.md

```diff
@@ -0,0 +1,131 @@
+# Surround
+
+*The framework used by the Applied Artificial Intelligence Institute (A<sup>2</sup>I<sup>2</sup>) to build, deploy and handover machine learning projects to clients.*
+
+**TLDR;** Surround is a framework developed by A<sup>2</sup>I<sup>2</sup> to take machine learning projects from concept through to production. Surround is designed to play nice with existing machine learning frameworks (Tensorflow, MXNet, PyTorch etc.) and cloud services (Google Cloud AI, SageMaker, Rekognition etc.) We designed Surround for software engineers and research engineers*, and is suitable for a range of deployment scenarios (offline, internal infrastructure, or in the cloud).
+
+A<sup>2</sup>I<sup>2</sup> is Australia's largest AI institute recently formed at Deakin University. Our team specialises in building early stage prototypes that are transferred to industry for evaluation and ongoing support. Due to the variety and number of projects we work on (more details [here](http://a2i2.ai)) we needed a flexible and consistent way to organise our machine learning projects. Surround was developed to help improve **reliability**, **maintainability**, **reproducibility** and **consistency** in our projects.
+
+Surround is now available to the community for feedback and is actively being developed [http://github.com/dstil/surround](http://github.com/dstil/surround).
+
+## Motivation
+
+**Huge effort is required to deploy an early prototype into a production environment**
+
+Typically, a project is first given to a research engineer to develop the initial solution (prepare the data, select an appropriate algorithm, train a model, evaluate the approach etc.) A software engineer is then tasked with preparing the scripts for integration and deployment into a client's environment. We found that many concerns of a production system such as how will configuration for different environments (development, staging and production) be handled, what is the format and location of the log files, will a queue or web interface be used etc. all impact how the initial solution is implemented. None of these concerns are paramount in the research engineers mind who typical come from a scientific background rather than an engineering background. Resolving these challenges ensuring the needs of the whole team (research engineers and software engineers) are satisfied is time consuming and must be repeated for each client project.
```

Comment:
We found concerns such as the handling of configuration for development, staging and production environments, the format and location of log files, whether a queue or web interface will be used, and so on, all influence the implementation of the initial solution. These concerns are not paramount to research engineers who do not typically come from an engineering background, but a scientific background. It is time consuming to repeatedly reconcile these differences in design and implement for each project.

Reply:
Thanks.